js/views/modals/purchase/Purchase.js

```diff
@@ -452,6 +452,7 @@ export default class extends BaseModal {
         singleSelect: true,
         selectFirst: true,
         radioStyle: true,
+        useCache: false,
```

Comment:
not how I read it:

![image](https://user-images.githubusercontent.com/794517/29592339-4da0bf16-8758-11e7-8cac-b1b971538808.png)